docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -722,7 +722,7 @@ public class PermissionsIdentityAugmentor implements SecurityIdentityAugmentor {
 Because `MediaLibrary` is the `TvLibrary` class parent, a user with the `admin` role is also permitted to modify `TvLibrary`.
 <2> You can add a permission checker through `io.quarkus.security.runtime.QuarkusSecurityIdentity.Builder#addPermissionChecker`.
 
-CAUTION: Annotation permissions do not work with the custom xref:security-customization.adoc#jaxrs-security-context[Custom Jakarta REST SecurityContext] because there are no permissions in `jakarta.ws.rs.core.SecurityContext`.
+CAUTION: Annotation permissions do not work with the custom xref:security-customization.adoc#jaxrs-security-context[Jakarta REST SecurityContext] because there are no permissions in `jakarta.ws.rs.core.SecurityContext`.
```

Comment:
I'd further clarify to "Annotation-based permissions".
"Annotation permissions" isn't very clear.

Reply:
I'd also remove the "the", because it's not one specific context, users can create any custom Jakarta REST SecurityContexts. So perhaps remove "the" and make it plural?

Reply:
GG